toolchains/java/README.md

```diff
@@ -75,6 +75,70 @@ build --tool_java_runtime_version=nixpkgs_java
 
 ##### Bazel 6
 
+#### with with [Bzlmod](https://bazel.build/versions/6.5.0/external/overview#bzlmod)
+
+Add the following to your `MODULE.bazel` file to depend on `rules_nixpkgs`, `rules_nixpkgs_java`, and nixpgks:
+```bzl
+bazel_dep(name = "rules_nixpkgs_core", version = "0.12.0")
+bazel_dep(name = "rules_nixpkgs_java", version = "0.12.0")
```

Comment:
i used a version here that does not exist yet because the instructions do not work with the current version as mentioned above.